packages/react-component-library/src/hooks/useDocumentClick.ts

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react'
+
+// workaround for undefined error in typescript, node.current might not be defined
```

Comment:
Did you manage to figure out what this was about?

Reply:
There is a TypeScript error when there is not an alternative for `node.current` being set. Whether this is the only option, I don't know.